crates/astria-composer/src/searcher/executor.rs

```diff
@@ -151,10 +165,15 @@ impl Executor {
             .await
             .wrap_err("failed getting initial nonce from sequencer")?;
         self.status.send_modify(|status| status.is_connected = true);
+
+        let mut block_timer = time::interval(Duration::from_millis(self.block_time));
```

Comment:
This should not be an interval timer. If you have submitted a tx right before the period completes you will attempt to send a (potentially empty) tx right after